src/main/java/uk/gov/hmcts/reform/roleassignment/config/SwaggerConfiguration.java

```diff
@@ -37,7 +37,8 @@ public Docket apiV2() {
             .globalOperationParameters(Arrays.asList(
                 headerServiceAuthorization(),
                 headerAuthorization(),
-                headerCorrelationId()
+                headerCorrelationId(),
+                headerAssignerId()
```

Comment:
If we are adding this under global param, it will be reflected to all other APIs as well..Can we limit it to only delete API?

Reply:
Done.